src/jit/lower.cpp

```diff
@@ -897,6 +969,16 @@ void Lowering::LowerNode(GenTreePtr* ppTree, Compiler::fgWalkData* data)
         LowerAdd(ppTree, data);
         break;
         
+    case GT_UDIV:
+    case GT_UMOD:
+        LowerUnsignedDivOrMod(*ppTree);
+        break;
+
+    case GT_DIV:
+    case GT_MOD:
+        LowerSignedDivOrMod(ppTree, data);
```

Comment:
A thought: Since these lowering transformations are more of an optimization than required for correctness, should we do this when not in minopts mode so that it provides an escape route if anyone hits a bug in these code paths? 

I don't have a strong opinion on this.